Library/Homebrew/requirements/codesign_requirement.rb

```diff
@@ -11,8 +11,8 @@ class CodesignRequirement < Requirement
 
   def initialize(tags)
     options = tags.shift
-    raise ArgumentError("CodesignRequirement requires an options Hash!") unless options.is_a?(Hash)
-    raise ArgumentError("CodesignRequirement requires an identity key!") unless options.key?(:identity)
+    raise ArgumentError, "CodesignRequirement requires an options Hash!" unless options.is_a?(Hash)
+    raise ArgumentError, "CodesignRequirement requires an identity key!" unless options.key?(:identity)
```

Comment:
As a historic footnote that may only interest me, this kind of error was an example in the very first public talk given on Sorbet: https://sorbet.run/talks/RubyKaigi2018/#/28